### ISSUE TITLE
[Feature] Export specific textures / models

### ISSUE THREAD

**Issue by fosspill** [fosspill/FFXIV_TexTools_CLI#6]

**Comment by shinnova** [comment]
Quick notes on how it should potentially work:

`ffmt mod export "fill in name here" --model all`
1. searches for user filled in name
2. multiple choices offered to user if string matches more than one item
3. export wanted data based on switches, maybe offer choices if nothing is defined?
4. export to directory defined in config? savedirectory?
5. model can be object, materials or both (all)
6. texture can be diffuse, normal, specular, multi, mask, colorset, icon or hq icon
7. need to figure out primary/secondary and part a/b/c/d differentiation
8. some items have special textures as well for special effects, but those have unique names

**Comment by fosspill** [comment]
Playing around with command ideas.

```
# -Mo = Model object
# - TnsMm = Texture normal specular multi mask
> ffmt mod export --name "Voeburtite Necklace" -Mo -TnsMm
Searching for Voeburtite Necklace...
Multiple items found:

.....
```

**Comment by CasualSuperman** [comment]
I'm not sure if this belongs in a separate issue, but I would also like to be able to extract a direct path - for example, `ui/uld/myccategory2_hr1.tex`

**Comment by lillelilje** [comment]
Indeed, that is what we've worked a bit on! A Wizard for guided exporting and some query for exporting specific path. Would be awesome if we could then also have some smart tabcompletion!
